subsys/net/lib/download_client/Kconfig

```diff
@@ -102,8 +102,8 @@ config DOWNLOAD_CLIENT_MAX_HOSTNAME_SIZE
 
 config DOWNLOAD_CLIENT_MAX_FILENAME_SIZE
 	int "Maximum filename length (stack)"
-	range 8 256
-	default 192
+	range 8 2048
+	default 1024
```

Comment:
Please do not increase the default for this specific use-case, incrase it in the affected project instead (especially that the filename buffer is allocated on a stack, so the project would need to increase the stack size as well.

Reply:
Fair enough, my goal was to avoid some error digging as I had to do when using S3 signed URLs which I would guess is a very common use case.

I changed the commit to default back to 192.